packages/suite/src/views/settings/general/ConnectLabelingProvider.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
I wonder why so many changes in visual components are needed. Wouldn't it make sense, to pull out at least some of the changes to a separate PR?

Reply:
I can do it but those visual improvements go hand in hand with the changes in logic. Since there is no prompt modal now and labeling is not enabled instantly, those changes provide more feedback to users regarding what is actually happening.